pkg/kubelet/cm/container_manager.go

```diff
@@ -60,6 +60,9 @@ type ContainerManager interface {
 	// GetNodeAllocatable returns the amount of compute resources that have to be reserved from scheduling.
 	GetNodeAllocatableReservation() v1.ResourceList
 
+	// GetNodeCapacity returns the amount of compute resources tracked by container manager available on the node.
+	GetCapacity() v1.ResourceList
+
```

Comment:
I'm not sure why the "container manager" is responsible for supplying all the information. If that's the case, maybe we can move `HasDedicatedImageFs` to here instead?